src/types/movie.ts

```diff
@@ -0,0 +1,13 @@
+export interface Movie {
```

Comment:
all interfaces should name, for example IMovie
IExample
IProps

Reply:
done